default/python/AI/InvasionAI.py

```diff
@@ -284,6 +286,12 @@ def evaluate_invasion_planet(planet_id, empire, secure_fleet_missions, verbose=T
         bld_tally += bval
         detail.append("%s: %d" % (bldType, bval))
 
+    tech_tally = 0
+    for unlocked_tech in AIDependencies.SPECIES_TECH_UNLOCKS.get(species_name, []):
+        if not tech_is_complete(unlocked_tech):
+            rp_cost = fo.getTech(unlocked_tech).researchCost(empire_id)
+            tech_tally += rp_cost * 10
```

Comment:
> We only extend length of the evaluate_planet by 8%.

The function definitely should be refactored at some point. It is way too long, has an extremely long if-else block and seems to redefine/reuse some variables from some hundred lines before... Not today and in this PR though.